src/components/app/nav.jsx

```diff
@@ -63,7 +63,9 @@ class AppNav extends Component {
       // }
       this.setState({info});
     }).catch(() => {
-      this.setState({info: null});
+      alert("Invalid user token");
```

Comment:
`Invalid user token` -> `GitHub token is invalid.`

Reply:
I'll change the message. Though I wasn't sure if the way I have handled this issue is a good user experience or not.

Reply:
It works but is not ideal. Instead of an alert, I would be happier to see an error message displayed on the modal. Besides, the modal shouldn't be closed and then reopened.